src/services/fluence-aurora-connector/modules/fluence_aurora_connector/src/jsonrpc/tests.rs

```diff
@@ -0,0 +1,87 @@
+use crate::{jsonrpc::{JsonRpcReq, get_logs::GetLogsReq}, chain::log::Log};
+
+use super::*;
+
+#[test]
+fn test_get_logs_request() {
+    let request = r#"
+{
+    "jsonrpc": "2.0",
+    "method": "eth_getLogs",
+    "params": [
+        {
+            "fromBlock": "0",
+            "toBlock": "latest",
+            "address": "0xD7Fa4fdaae7b69A2b7B87A860fFbDB8232310a43",
```

Comment:
not a big deal now but when a chain change comes, this is all gonna break. todo: maybe put params in env fil and use chainid to load for tests?

Reply:
Why is it gonna break? That's all hardcoded, there's no networking in these tests, it just tests ser/de